README.md

```diff
@@ -189,3 +199,16 @@ WARN[2017-07-11T12:24:32+02:00] job took too long to run: it should have started
   [cronexpr]: https://github.com/gorhill/cronexpr
   [releases]: https://github.com/aptible/supercronic/releases
   [glide-install]: https://github.com/Masterminds/glide#install
+  
+  
+## Contributing ##
+
+[comment]: <> (Should we provide some language welcoming contributions and explaining how someone might go about contributing?)
+
+## Author ##
+
+[Aptible](https://www.aptible.com)
+
+## License ##
+
+[comment]: <> (Should we mention that this is open source, using something like Apache or MIT?)
```

Comment:
It's indeed MIT: https://github.com/aptible/supercronic/blob/master/LICENSE.md

GitHub actually autodetects this, but I think it's also worth noting here in the README.

![image](https://user-images.githubusercontent.com/156030/28173314-16d5eeb8-67bc-11e7-9cda-3077c32c9988.png)